dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts

```diff
@@ -11,19 +12,28 @@ import {WidgetService} from '../../shared/widget.service';
 export class PlatformStatusWidgetComponent extends Widget {
 
   version: string = 'N/A';
+  titleUrl: string;
+  description: string;
```

Comment:
`titleUrl` and `description` should be placed in `Widget` super class, as every widget has these attributes.